src/controllers/settings.ts

```diff
@@ -0,0 +1,82 @@
+import { Body, Controller, Ctx, Get, Post, Put, Query } from 'amala'
+import { Context } from 'vm'
+import { Settings, SettingsModel, findOneOrCreate } from '@/models/Settings'
+import { TokenModel } from '@/models/Token'
+import { badRequest, internal } from '@hapi/boom'
+
+@Controller('/settings')
+export default class SettingsController {
+  @Post('/')
+  async addSettings(
+    @Body({ required: true })
+    { allPostsEnabled, hotPostsEnabled, repliesEnabled, token }: Settings,
+    @Ctx() ctx: Context
+  ) {
+    try {
+      const tokenRecord = await TokenModel.findOne({
+        token,
```

Comment:
We can remove `,` from here to make it one-line

Reply:
Nope, The linter don't allow that